### PR TITLE
Fix broken image and typo

### LINKED ISSUE #1323: Python Anywhere API key instructions are incorrect and image is missing
### Issue description

On the Installation page, it says I need to create an account on Python Anywhere. That worked OK.

Then it says I need to create an API key. The instructions say:

When you've signed up for PythonAnywhere, you'll be taken to your dashboard. Find the link near the top right to your "Accounts" page, then select the tab named "API token", and hit the button that says "Create new API token".

The link says "Account", not "Accounts".

Also, there is an image file on the page with an Alt text that says "The API token tab on the Accounts page" goes to
https://tutorial.djangogirls.org/en/installation/images/pythonanywhere_create_api_token.png
but that image is missing.

(Sorry if I'm making too many issues. I've signed up to be a Coach at an upcoming workshop in Spokane and I'm working through the tutorial, making issues as I go if things are incorrect. Hope that is OK! I will probably start making pull requests to fix them once I get through the tutorial.)

### PR DESCRIPTION
**Pull request by jhodgdon-drp**
Fixes #1323

### CODE REVIEW

File: en/deploy/signup_pythonanywhere.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ Sign up for a free "Beginner" account on PythonAnywhere:
 
 ## Creating a PythonAnywhere API token
 
-This is something you only need to do once.  When you've signed up for PythonAnywhere, you'll be taken to your dashboard.  Find the link near the top right to your "Accounts" page, then select the tab named "API token", and hit the button that says "Create new API token".
-
-<img src="images/pythonanywhere_create_api_token.png" alt="The API token tab on the Accounts page" />
+This is something you only need to do once.  When you've signed up for PythonAnywhere, you'll be taken to your dashboard.  Find the link near the top right to your "Account" page, then select the tab named "API token", and hit the button that says "Create new API token".
 
+![The API token tab on the Accounts page](../deploy/images/pythonanywhere_create_api_token.png)
```
[the remark this answers is not in the document]
Comment (jhodgdon-drp) [reply en/deploy/signup_pythonanywhere.md]: Right, sorry, it's Account. Fixed in one place, missed that one!

[the remark this answers is not in the document]
Comment (jhodgdon-drp) [reply en/deploy/signup_pythonanywhere.md]: Actually we should look at the screenshot and see if it needs an update... will report back shortly.

[the remark this answers is not in the document]
Comment (jhodgdon-drp) [reply en/deploy/signup_pythonanywhere.md]: Image is OK. It is pared down and doesn't show the Account link at all. I'll edit the PR...

**Comment by ekohl** [comment]
@jhodgdon-drp I've changed the PR message so it automatically closes the issue when this is merged.

**Pull request merged**